client/settings/patch-starcraft.js

```diff
@@ -89,11 +95,23 @@ async function patchFile(dirPath, outPath, filename, validHashes) {
   }
 }
 
+async function copyLocalDll(dirPath, downgradePath) {
+  const matches = await globAsync(`${dirPath}/locales/*/local.dll`)
+  if (matches.length < 1) {
+    throw new Error("local.dll doesn't exist in StarCraft directory")
```

Comment:
No, this isn't the only possible case. There are 2 cases:
- local.dll is in the root folder
- local.dll is in the locales/*/ folder

The first will happen for all version < 1.19, the latter will happen for 1.19 or later. We support all such versions, so you need to handle both cases.